packages/coinstac-headless-client/src/parse-csv-files.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
```

Comment:
I wonder if we should move this, and the pipeline mapping, to some sort of common utility package. I mean there is coinstac-common, which is basically unused right now